content/HTTPS is not enough - Pablo Blanco.md

```diff
@@ -0,0 +1,218 @@
+middle SSLstrip"** and **"session hijacking"** attacks.
+
+Let’s suppose this scenario, an user of our application log into a free WiFi
+access point at an airport and start surfing the web, visiting our online
+banking service to make some transactions, and pay a couple of bills.
+Unfortunately, that access point can be sniffed by an attacker, and it’s really
+easy to intercept all the original HTTP requests and redirect them to a clone of
+the bank's site instead of the real thing. Then, the user’s data is exposed to
+the attacker.
+
+Well maybe you are thinking, it is not supposed that anyone access to its bank
+account on a public Wifi network, and you are completely right, but the same
+scenario can be considered if someone hacks into our **private Wifi network**
+too.
+
+This kind of man in the middle attack is basically performed by the following
+steps:
+
+![White or black hat?](images/https-not-enough-mitm.png)
+
+1.  User asks for `https://somedomain.com`
+2.  MITM intercept that, and ask for `https://somedomain.com` to the Server
+3.  The server responds to MITM `https://somedomain.com`
+4.  MITM responds to the User with `http://somedomain.com`
+
+After that, the attacker can see plain data over the requests made by the user.
+
+## How can we mitigate HTTPS vulnerabilities?
+
+Whenever a browser requests a page from a web server, the server answers to the
+browser with the content along with some useful information, contained in
+**headers**. Many SSL-related vulnerabilities can be mitigated with a special
+type of headers, called security headers that can tell browsers how to treat
+content on a web page such as “whitelist certain types of content from trusted
+sites” or “only load this site over HTTPS”, for this particular vulnerabilities
+there are 2 types of headers we can use, **HSTS** and **CSP**.
+
+### HTTP Strict Transport Security (HSTS)
+
+The HTTP Strict-Transport-Security response header, lets a website tell browsers
+that it should only be accessed using HTTPS strictly, instead of using HTTP and
+should automatically convert all attempts to access the site using HTTP to HTTPS
+requests instead.
+
+This header is specified by [RFC6797](https://tools.ietf.org/html/rfc6797) and
+contains three directives:
+
+- **max-age** (mandatory): time in seconds, that the browser should remember
+  that a site is only to be accessed using HTTPS.
+
+- **includesubdomains** (optional): indicates if this rule applies to all of the
+  site's subdomains as well.
+
+- **preload** (optional): indicates that the site meets requirements for HSTS
+  preloading and is on the HSTS preload list or has applied for it.
+
+Once the browser has set up HSTS for a site, all communication with that site
+will use HTTPS, which **eliminates the SSL stripping threat** and guarantees
+secure transmission.
+
+#### Keep browsers updated
+
+Everything looks fine still here, but the first time an user visits an unknown
```

Comment:
What do you think about `Everything looks fine so far,` instead of `Everything looks fine still here,`